### PR TITLE
Added com.visualstudio.code.insiders.tool.podman manifest

### PR DESCRIPTION
**Pull request by rtmor**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.visualstudio.code.insiders.tool.podman

File: com.visualstudio.code.insiders.tool.podman.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  - name: bundle-setup
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 -t $FLATPAK_DEST/share/metainfo $FLATPAK_ID.metainfo.xml
+      - appstream-compose --basename=$FLATPAK_ID --prefix=$FLATPAK_DEST --origin=flatpak
+        $FLATPAK_ID
+    sources:
+      - type: file
+        path: com.visualstudio.code.insiders.tool.podman.metainfo.xml
```
Comment (hfiguiere) [inline com.visualstudio.code.insiders.tool.podman.yml:45]: this file is missing

**Comment by hfiguiere** [comment]
also you still try to build an aarch64 package.

**Comment by gasinvein** [comment]
You don't need a separate extension for the insiders variant flatpak. Instead, the app should fix their manifest to have the `com.visualstudio.code.tool` extension point instead of `com.visualstudio.code.insiders.tool`.

**Pull request closed without merging**